docs/root/intro/arch_overview/upstream/circuit_breaking.rst

```diff
@@ -13,7 +13,15 @@ configure and code each application independently. Envoy supports various types
 
 * **Cluster maximum connections**: The maximum number of connections that Envoy will establish to
   all hosts in an upstream cluster. If this circuit breaker overflows the :ref:`upstream_cx_overflow
-  <config_cluster_manager_cluster_stats>` counter for the cluster will increment.
+  <config_cluster_manager_cluster_stats>` counter for the cluster will increment. All connections,
+  whether active or draining, count against this limit. Even if this circuit breaker has overflowed,
+  Envoy will ensure that a host selected by cluster load balancing has at least one connection
+  allocated. This has the implication that the :ref:`upstream_cx_active
+  <config_cluster_manager_cluster_stats>` count for a cluster may be higher than the cluster maximum
+  connection circuit breaker, with an upper bound of
+  `cluster maximum connections + (number of endpoints in a cluster) * (connection pools for the
+  cluster)`. See :ref:`connection pooling <arch_overview_conn_pool_how_many>` for details on how many
```

Comment:
The how many docs don't talk about workers, and the circuit breaker limits are shared across workers. I think it would be worth it to clarify this?

Also, it's somewhat implied in the above, but it might also be worth calling out the eventually consistent circuit breaker impl, meaning that across workers, it is possible for there to be a race and the value to go higher (I guess technically it can get <num workers> above the set limit I think)? I'm not sure if we explain this elsewhere in the page but worth calling out.